app/src/components/adminConsole/currentCup/AllCurrentCupsDisplay.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import Cup from "../../../types/Cup";
+import { Entity } from "../../../types/Entity";
+import AdminCurrentSnapsDisplay from "./AdminCurrentSnapsDisplay";
+import { CupNameDisplay, MessageDisplay } from "../AdminConsoleStyles";
+
+const AllCurrentCupsDisplay: React.FunctionComponent = (props: {
+    cups: Entity<Cup>[];
+    updateCups: () => void;
+    setCups;
+}) => {
+    const cupsList = props.cups.map((cup) => {
+        if (cup.isPublished) {
+            return null;
```

Comment:
This filtering should really happen a level up, not mixed in with the display logic.